src/main/java/toast/specialMobs/entity/spider/EntityMotherSpider.java

```diff
@@ -47,11 +55,10 @@ public void adjustTypeAttributes() {
     /// Called when this entity is killed.
     @Override
     protected void dropFewItems(boolean hit, int looting) {
+        // ALL CHANGES IN HERE MUST BE ALSO MADE IN provideDropsInformation
         super.dropFewItems(hit, looting);
         if (hit && (this.rand.nextInt(3) == 0 || this.rand.nextInt(1 + looting) > 0)) {
-            this.entityDropItem(
-                    new ItemStack(Items.spawn_egg, 1, EntityList.getEntityID(new EntitySpider(this.worldObj))),
-                    0.0F);
+            this.entityDropItem(new ItemStack(Items.spawn_egg, 1, 52 /* EntitySpider */), 0.0F);
```

Comment:
I understand that creating a new entity whenever you need to just retrieve its id is wasteful. But is there any other way to get that *magic* number?

Reply:
I changed it to `net.minecraft.entity.EntityList#classToIDMapping`